<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="3571" started-at="2020-11-19T19:27:34 IST" finished-at="2020-11-19T19:27:38 IST">
    <groups>
    </groups>
    <test name="Default test" duration-ms="3571" started-at="2020-11-19T19:27:34 IST" finished-at="2020-11-19T19:27:38 IST">
      <class name="Mccm.Pega.DMPRealTime.PegaTest.DMPRTPNBAMobHashTest">
        <test-method status="FAIL" signature="VerifyDMPRTPNBAMobHashedAPIcallSuccessfully()[pri:0, instance:Mccm.Pega.DMPRealTime.PegaTest.DMPRTPNBAMobHashTest@4671e53b]" name="VerifyDMPRTPNBAMobHashedAPIcallSuccessfully" duration-ms="3511" started-at="2020-11-19T19:27:34 IST" finished-at="2020-11-19T19:27:38 IST">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Status code is not 200 , expected [200] but found [400]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Status code is not 200 , expected [200] but found [400]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertEqualsImpl(Assert.java:136)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:839)
at Mccm.Pega.DMPRealTime.PegaTest.DMPRTPNBAMobHashTest.VerifyDMPRTPNBAMobHashedAPIcallSuccessfully(DMPRTPNBAMobHashTest.java:92)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:135)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:821)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:767)
at org.testng.TestRunner.run(TestRunner.java:588)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1214)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1136)
at org.testng.TestNG.runSuites(TestNG.java:1066)
at org.testng.TestNG.run(TestNG.java:1034)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- VerifyDMPRTPNBAMobHashedAPIcallSuccessfully -->
      </class> <!-- Mccm.Pega.DMPRealTime.PegaTest.DMPRTPNBAMobHashTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
